### PR TITLE
ADEN-6759 Adding Wikia key-vals to Rubicon Video

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Review by kvas-damian — DISMISSED** [review]
Please make sure rubicon isn't returning 400s before merging.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js [hunk 1]
```diff
@@ -27,12 +28,48 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.rubicon', [
...
+			adUnit,
+			bidParams;
+
+		bidParams = {
+			accountId: rubiconAccountId,
+			siteId: rubiconSiteId,
+			zoneId: config.zoneId,
+			name: slotName,
+			position: config.position,
+			inventory: targeting,
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js]: In display we're using `config.targeting` by default. Do you think we should use it here as well?

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js [hunk 2]
```diff
@@ -27,12 +28,48 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.rubicon', [
...
+
+		return {
+			pos: slotName,
+			src: provider,
+			s0: adLogicZoneParams.getSite(),
+			s1: s1,
+			s2: adLogicZoneParams.getPageType(),
+			lang: adLogicZoneParams.getLanguage()
+		};
+	}
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js]: This method is a copy&past from `rubiconDisplay.js`. Can we extract it?

**Comment by kvas-damian** [comment]
Maybe we could try pass arrays as a value. See an example here: https://github.com/prebid/Prebid.js/pull/950

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js]: Good idea. Refactored.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**